internal/app/wwctl/kernel/imprt/main.go

```diff
@@ -14,58 +14,57 @@ import (
 )
 
 func CobraRunE(cmd *cobra.Command, args []string) error {
-	for _, arg := range args {
-		// Checking if container flag was set, then overwriting OptRoot
-		if OptContainer != "" {
-			if container.ValidSource(OptContainer) {
-				OptRoot = container.RootFsDir(OptContainer)
-			} else {
-				wwlog.Printf(wwlog.ERROR, " %s is not a valid container", OptContainer)
-				os.Exit(1)
-			}
-		}
-		output, err := kernel.Build(arg, OptRoot)
-		if err != nil {
-			wwlog.Printf(wwlog.ERROR, "Failed building kernel: %s\n", err)
-			os.Exit(1)
+
+	// Checking if container flag was set, then overwriting OptRoot
+	kernelVersion := args[0]
+	kernelName := args[1]
```

Comment:
This is going to cause a panic if args[1] is undefined.

Reply:
Should be fixed now.